README.md

```diff
@@ -6,5 +6,9 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework that was written in ruby. It is used to create a back end structure for a web app to interact with.
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
This is partially right, but there are a couple of errors. For instance, the request doesn't get passed along from one component to the next. The Server receives the request, and based on the content of that request, it triggers a Controller action (that controller will have access to some of the data from the request, though). The Controller then calls a method on the Model, telling it to perform some CRUD action; the Model returns something as a result (probably data), and the Controller then takes that data and constructs a View, which gets sent back to the Server.